program/Cargo.toml

```diff
@@ -35,3 +35,11 @@ crate-type = ["cdylib", "lib"]
 
 [build-dependencies]
 rustc_version = "0.4"
+
+[lints.rust.unexpected_cfgs]
+level = "warn"
+check-cfg = [
+    'cfg(target_os, values("solana"))',
+    'cfg(feature, values("min_specialization"))',
+    'cfg(RUSTC_WITH_SPECIALIZATION)',
```

Comment:
I'm surprised that we need these here, considering the monorepo doesn't. Do you know why that is?

Reply:
I'm not entirely sure, but it seems like the monorepo originally had build scripts like the one here, and it was to configure specialization to somehow ensure `frozen-abi` was only evaluated with nightly?

Either way, after Kevin feature-flagged the whole monorepo with `frozen-abi`, it appears the requirement has been removed. It now just depends on the feature instead. We can explore just removing the build script and following that pattern instead.

Reply:
Ah yeah, we'll have to do https://github.com/anza-xyz/agave/pull/2911 for this repo too

Reply:
Thanks for mopping that up!